sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1551,6 +1551,20 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     FillSliceVertex(vertex, recslc);
     FillSliceCRUMBS(slcCRUMBS, recslc);
 
+    // Fill flash match time wrt beam spill
+    if(isRealData) {
+      // If no trigger just ignore this variable
+      if(fSRTrigger.size() > 0) {
+        // fSRTrigger is vector of events, but we should only be looking at
+        // one event at a time. So just use first trigger in the vector.
```

Comment:
I used the word event here since that's what Jacob Z used in the comment describing this in SRHeader.h. So yes, if this contains more than one entry something has gone very wrong which is why I just use the first element.